tests/Responses/Expected/order/order_show_response.json

```diff
@@ -5,7 +5,7 @@
     "adjustments": [],
     "adjustments_total": 0,
     "total": 0,
-    "state": "cart",
+    "state": "new",
```

Comment:
The `checkoutState` field is redundant for already placed order. Btw. it shouldn not be `cart`, but `completed` does not seem correct also...